content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,140 @@
+* Even more integrations have been built by our open source community, in [integrations extras][4].
+* You can build your own integration using [Datadog API][5].
+* The Agent is [open source][6] and you can instrument your own if you'd like.
+* All incoming data is treated the same throughout Datadog.
+
+It's best to start monitoring your projects as early in the development process as possible, but you can start at any stage.
+
+### Key terms
+
+**parameters** - Use the parameters in the `conf.yaml` file to control accesses between your integration data source and the Agent. The individual integrations `conf.yaml.example` file has all of the required and not required parameters listed. Define all of the needed parameters for all of your integrations in a single `conf.yaml` file - if you have more than one file, the Agent only uses the alphabetically first file.
+
+**conf.yaml** - You can find the `conf.yaml` in the `conf.d/` folder at the root of your [Agent's configuration directory][7]. Use this file to connect integrations to your system, as well as configure their settings. Nonrequired parameters are usually commented out.
+
+**instance** - You define and map the instance of whatever you are monitoring in the `conf.yaml` file. For example, in the [`http_check` integration][8], you're defining the name associated with the instance of the HTTP endpoint you are monitoring up and downtime. You can monitor multiple instances in the same integration, and you do that by defining all of the instances in the `conf.yaml` file.
+
+**metrics** - The list of what is collected from your system by each integration. You can find the metrics for each integration in that integrations `metadata.csv` file. For more information about metrics, see the [Metrics][9] developer page. You can also set up [custom metrics][10], so if the integration doesn't offer a metric out of the box, you can usually add it.
+
+**metadata.csv** - The file that lists and stores the metrics collected by each integration.
+
+**custom check** - If you have a unique system that you want to monitor, or if you're going to expand the metrics already sent by an integration, you can build a [custom check][11] to define and send metrics to Datadog. However, if you want to monitor a generally available application, public service, or an open source project, and we don't have an integration that already exists for it, you should consider [building a new integration][1] instead of a custom check.
+
+**tagging** - [Tags][12] are a way to add customization to metrics so that you can filter and visualize them in the most useful way to you.
+
+**logging** - If the system you are monitoring has logs, you can customize the logs you are sending to Datadog and use our [Logging Management solution][13] to manage and analyze them.
+
+## Setting up an integration
+
+The Datadog Agent package includes most integrations, so you don't need to download each individually.
```

Comment:
This sentence is a little bit too fuzzy, Datadog Agent package includes integrations officially supported by Datadog, they can be found here: https://github.com/DataDog/integrations-core, community based integration can also be used but are not packaged with the Agent, for more information on how to install them or how to manage officialy supported integrations they can refer to this guide: https://docs.datadoghq.com/agent/guide/integration-management/